exec/java-exec/src/main/java/org/apache/drill/exec/coord/zk/ZookeeperClient.java

```diff
@@ -133,13 +137,50 @@ public boolean hasPath(final String path, final boolean consistent) {
    * the check is eventually consistent.
    *
    * @param path  target path
+   * @param consistent consistency flag
```

Comment:
Perhaps explan this a bit more. The comment line above says "the check is eventually consistent." How does the consistent flag change the semantics of this check?